src/features/conversations/components/ConversationPanel.module.css

```
@@ -87,6 +89,25 @@
   }
 }
 
+.systemMessage {
+  align-self: center;
+  max-width: 80%;
+  padding: 1rem;
+
+  background: #303438ad;
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)